packages/cms-lib/lib/config.js

```diff
@@ -109,7 +121,7 @@ const getOrderedConfig = unorderedConfig => {
     defaultMode,
     httpTimeout,
     allowsUsageTracking,
-    portals,
+    portals: portals.map(getOrderedPortal),
```

Comment:
In general, I am not super comfortable with auto-formatting data that is outside the context of the command.